src/controller/userController.js

```diff
@@ -28,10 +28,16 @@ const addUser = async (req, res, next) => {
 const updateUser = async (req, res, next) => {
   const { id } = req.params;
   try {
-    const updatedUser = await User.findByIdAndUpdate(id, req.body, {
-      new: true,
-      useFindAndModify: false,
-    });
+    const updatedUser = await User.findByIdAndUpdate(
+      id,
+      req.body,
+      {
+        new: true,
+        useFindAndModify: false,
+      },
+      (err, model) => next(err)
+    );
+
```

Comment:
tests in insomnia are showing that i cant update a user after these change , server error 404 - cannot find requested ressource.

Reply:
True,  I am on it, try again please.

Reply:
nope still not

Reply:
now it works